compute/cker/src/train/SoftMax.test.cc

```diff
@@ -65,14 +65,14 @@ TEST(CKer_Optimizer, SoftMaxGrad)
       -4.48784290e-05, 4.90605867e-03,  -1.53427655e-08, -7.02857016e-04, -4.81329140e-09,
       -5.15774553e-04, -1.04579225e-04, -7.00410377e-03, -1.29717840e-02, -8.63838721e-11,
       -7.12137857e-05, 2.13432343e-02,  -4.23775872e-06, -5.91166379e-04, -8.03746891e-05};
-    std::vector<float> grad = {0, 0, 0, 0, 0, 0, 0, 0, 0, 0};
+    std::vector<float> grad = {0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0};
```

Comment:
Just for recording.

There are 20 zeros. 20 is the flat size of `shape` (2x10).
```suggestion
    std::vector<float> grad(20);
```